pfunk/__main__.py

```diff
@@ -297,7 +298,13 @@ def investigate(args):
         pfunk.tests.plot(name, args.show)
 
 
+class CleanFileAction(argparse.Action):
+    def __call__(self, parser, namespace, values, option_string=None):
+        setattr(namespace, self.dest, pfunk.clean_filename(values))
+
+
 def main():
+    default_database_path = "./test_results.db"
```

Comment:
Probably better to store in the main module somewhere?

Reply:
Good spot, thanks, I've moved it to a constant in `__init__.py`